.circleci/config.yml

```diff
@@ -765,7 +766,7 @@ jobs:
     circleci_ip_ranges: true
     docker:
         - image: <<pipeline.parameters.default_docker_image>>
-    resource_class: 2xlarge
+    resource_class: xlarge
```

Comment:
Nothing blocking just curious, how did we arrive at the conclusion of downgrading the runner? (any indicators of under-utilisation which I am probably unaware of?)

Reply:
I took a look at the resource usage of this job and it wasn't using anywhere close to the max:

<img width="1105" alt="Screenshot 2025-02-06 at 9 39 42 AM" src="https://github.com/user-attachments/assets/2ae06a8e-0862-4025-ab20-e4a926da84b5" />
<img width="1087" alt="Screenshot 2025-02-06 at 9 39 58 AM" src="https://github.com/user-attachments/assets/9fb3880c-2ee7-4514-b564-2316c70d2880" />

Reply:
Fab! Thank you.